game.py

```diff
@@ -90,11 +90,9 @@ def readCommand( argv ):
     args['peasant1Type'] = options.peasantAgent1
     args['peasant2Type'] = options.peasantAgent2
     args['numGames'] = options.numGames
-    args['timeout'] = options.timeout
 
     return args
 
-
 def runGames(landlordType, peasant1Type, peasant2Type, numGames, timeout):
     """
```

Comment:
Instead of setting `numGames` each time, would it be better to set an exit argument? Such as ending the game once `exit` is typed in the terminal and keeping the game running forever if `exit` has yet typed in.

Reply:
I see what you're saying, but my intention for having `numGames` be an argument is so that we can run a bunch of trials of particular agents playing (ex: have an expectiminimax landlord play against two peasant random agents 100 times) and collect statistics to see how good our agents are. Might be something to look into in the future though